clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -232,6 +232,9 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
       return getConstPtrAttr(ptrTy, 0);
     if (auto structTy = ty.dyn_cast<mlir::cir::StructType>())
       return getZeroAttr(structTy);
+    if (auto boolTy = ty.dyn_cast<mlir::cir::BoolType>()) {
```

Comment:
Since `boolTy` is not used, `ty.isa<mlir::cir::BoolType>()` might be better

Reply:
fixed